examples/node/dynamic_codegen/route_guide/route_guide_server.js

```diff
@@ -117,6 +117,7 @@ function getDistance(start, end) {
   var lat2 = toRadians(end.latitude / COORD_FACTOR);
   var lon1 = toRadians(start.longitude / COORD_FACTOR);
   var lon2 = toRadians(end.longitude / COORD_FACTOR);
+  var dlon = end.longitude - start.longitude;
```

Comment:
We already have "deltalon" declared below. But you're right that there's a problem: occurences of dlon should be replaced with "deltalon".